fe/fe-core/src/main/java/com/starrocks/privilege/DbPEntryObject.java

```diff
@@ -127,7 +125,7 @@ public boolean match(Object obj) {
             return this.catalogId == other.catalogId;
         }
         return this.catalogId == other.catalogId &&
-                Objects.equals(getCompatibleDbUUID(this.uuid), getCompatibleDbUUID(other.uuid));
+                Objects.equals(Catalog.getCompatibleDbUUID(this.uuid), Catalog.getCompatibleDbUUID(other.uuid));
```

Comment:
What the difference between these two functions?

Reply:
No actual difference in this place. It is more appropriate to use `Catalog.getCompatibleDbUUID` since this function handles databases from both internal catalog and external catalog.